src/main/java/de/azapps/kafkabackup/sink/BackupSinkConfig.java

```diff
@@ -103,4 +107,7 @@ Boolean pathStyleAccessEnabled() {
     String region() {
         return getString(AWS_S3_REGION);
     }
+    Long consumerGroupsSyncInterval() {
+        return getLong(CONSUMER_GROUPS_SYNC_MAX_AGE);
```

Comment:
It seems like we use 3 different terms for the same thing here.

- `offsets.consumer.list.max.age.ms`
- `CONSUMER_GROUPS_SYNC_MAX_AGE`
- `consumerGroupsSyncInterval`
- `DEFAULT_SYNC_INTERVAL`

I suggest hamonizing these names to make it easier to understand. Perhaps use

- `consumer.groups.sync.max.age.ms`
- `CONSUMER_GROUPS_SYNC_MAX_AGE_MS`
- `consumerGroupsSyncMaxAgeMs()`

or similar?